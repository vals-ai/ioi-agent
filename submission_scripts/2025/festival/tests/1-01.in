version https://git-lfs.github.com/spec/v1
oid sha256:c9d16b5b0fb11672a5e76f63e574fb8d8021947dd5f8f78da0134226c1af86a6
size 2378186
