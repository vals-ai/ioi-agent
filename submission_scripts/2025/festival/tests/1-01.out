version https://git-lfs.github.com/spec/v1
oid sha256:be6d1b44c6b43da8525ab5f1bc9be7f6c544cd0504ef508163d5df5f01fdb071
size 4087
